migrations/0003-databaseuser.sql

```diff
@@ -0,0 +1,9 @@
+-- -----------------------------------------------------
+-- Table `animalkeeping`.`DatabaseUserType`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `animal_keeping`.`DatabaseUserType` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(200) NOT NULL,
+  `privileges` VARCHAR(500) NOT NULL,
+  PRIMARY KEY (`id`)) 
+ENGINE = InnoDB;
```

Comment:
>the standard UserTypes should be part of the script.
like the following... 

Done. Also changed 0001-initial.sql to incorporate the new column in census_person.